controllers/featureflagconfiguration_controller.go

```diff
@@ -47,10 +49,41 @@ type FeatureFlagConfigurationReconciler struct {
 // For more details, check Reconcile and its Result here:
 // - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.11.0/pkg/reconcile
 func (r *FeatureFlagConfigurationReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	_ = log.FromContext(ctx)
+	logger := log.FromContext(ctx)
 
-	// TODO(user): your logic here
+	podList := &v1.PodList{}
 
+	// I'd like to be able to do some field-matching with the ListOptions so we don't have to iterate,
+	// but they can't seem to be used for annotations.
+	// we may want our webhook to also add labels for easy querying?
+	if err := r.List(ctx, podList); err != nil {
+		return ctrl.Result{}, client.IgnoreNotFound(err)
+	}
+
+	// find any pods that are associated with this CR
+	for _, pod := range podList.Items {
+if val, ok := pod.ObjectMeta.Annotations["openfeature.dev/featureflagconfiguration"]; ok {
+if val == req.Name {
+			configMapList := &v1.ConfigMapList{}
+			
+			// query configMaps
+			r.List(ctx, configMapList) 
+			for _, configMap := range configMapList.Items {
+				// find the configMap matching the pod name (this is how our webhook names them for now, might want something else long-term)
+				if (configMap.Name == pod.Name) {
+
+					// get the new contents by querying our CR based on the request data.
```

Comment:
We need a way of actually comparing the contents of the configmap config.yaml against the FeatureFlagConfiguration Spec internals

Reply:
Created: https://github.com/open-feature/open-feature-operator/issues/20